packages/cli/api-importers/commons/src/FernDefnitionBuilder.ts

```diff
@@ -92,28 +92,25 @@ export class FernDefinitionBuilderImpl implements FernDefinitionBuilder {
     private packageMarkerFile: RawSchemas.PackageMarkerFileSchema = {};
     private basePath: string | undefined = undefined;
 
-    public constructor(
-        ir: OpenApiIntermediateRepresentation,
-        private readonly modifyBasePaths: boolean,
-        public readonly enableUniqueErrorsPerEndpoint: boolean
-    ) {
+    public constructor(public readonly enableUniqueErrorsPerEndpoint: boolean) {
         this.root = new FernDefinitionDirectory();
         this.rootApiFile = {
             name: "api",
             "error-discrimination": {
                 strategy: "status-code"
             }
         };
-        if (ir.title != null) {
-            this.rootApiFile["display-name"] = ir.title;
-        }
```

Comment:
Is this intended? Why not include `display-name` anymore? 

It looks like we have the new `setAPIDisplayName`, but that sets the root API file's `name`, not `display-name`.